com.github.limads.Queries.json

```diff
@@ -0,0 +1,59 @@
+{
+    "app-id" : "com.github.limads.Queries",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "42",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "queries",
+    "finish-args" : [
+        "--share=network",
+        "--device=dri",
+        "--socket=wayland",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--filesystem=home"
```

Comment:
why do you need home acces?

Reply:
The application requires access to files at paths specified by the user (SSL certificates) and there is no way to anticipate where the user is going to store them. Plus, a lot of the interactions with scripts are not based on file chooser dialogs, therefore the home access.

Reply:
If the App uses a Gtk Filepicker, the App will get access to the files the User picked, so you don't need home access.

Reply:
There are certain places it doesn't use a file chooser dialog.